src/setupTests.js

```diff
@@ -1,16 +1,19 @@
+import '@testing-library/jest-dom';
+import '@testing-library/jest-dom/vitest';
+import 'babel-polyfill';
+import 'jsdom-worker-fix';
 import Enzyme from 'enzyme';
 import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
-import 'jsdom-worker-fix';
 import { act } from '@testing-library/react';
-import '@testing-library/jest-dom';
-import 'babel-polyfill';
 import ResizeObserverPolyfill from 'resize-observer-polyfill';
+import { cleanup } from '@testing-library/react';
```

Comment:
We have 2 imports from the same package `@testing-library/react`. Consolidate them into:
`import { act, cleanup } from '@testing-library/react';`